README.md

```diff
@@ -70,22 +70,30 @@ Use `docker build [...] --build-arg X=Y` to change the defaults (see [Docker doc
 ## Building
 
 ---
-1. Build the docker image (example tagged with `latest`)
+1. Copy the `env.example` file to `.env` and modify values (see the comments in the `env.example` for more details), or just do:
+
+```bash
+echo "TB_CHAINS_MAINNET_RPCPROVIDER=http://yourRpcEndpoint:port" > .env
+# Make sure to provide a valid RPC endpoint that exposes both an archive node and the trace_ namespace
+```
+
+2. Build the docker image (example tagged with `latest`)
 
   ```bash
   docker build ./build --tag=trueblocks-core:latest
   ```
 
-2. Run the container
+3. Run the container
 
   ```bash
   # By default, both scraper and chifra serve (API server) are started
   docker run \
     --name trueblocks-core \
     --env-file ./.env \
     --publish 8080:8080 \
-    --mount type=bind,source=/Volumes/IndexCache/trueblocks/cache,target=/cache \
```

Comment:
It's my understanding that one can accomplish the identical behaviour with `--mount` as one can with `--volume` and that, for the most part, `--volume` is shorthand for `--mount` which provides additional features.

1) Is that true,
2) Does the suggested change modify anything behaviorally or is it just syntactic?
3) If it's only syntactic, then why the change (other than for clarity)?

Reply:
1. Yes other than one behavior.
2. The `-v` flag always creates the host file/directory that is being mounted. The `--mount` flag would require you to make the `/Volumes/IndexCache/trueblocks/cache` and `/Volumes/IndexCache/trueblocks/unchained` directories beforehand.
3. If the change is not made, it would make sense to specify the requirement of making the two directories before running the container.